.github/CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+app/wasm/**/* @nicolaslara @czarcas7ic
+x/concentrated-liquidity/**/* @czarcas7ic @p0mvn
+x/downtime-detector/**/* @nicolaslara
+x/epochs/**/* @mattverse @nicolaslara
+x/gamm/**/* @mattverse @czarcas7ic @p0mvn
+x/incentives/**/* @mattverse @p0mvn
+x/lockup/**/* @mattverse @stackman27 @czarcas7ic
+x/mint/**/* @p0mvn
+x/pool-incentives/**/* @czarcas7ic @p0mvn
+x/poolmanager/**/* @czarcas7ic @p0mvn @mattverse
+x/protorev/**/* @p0mvn
+x/superfluid/**/* @mattverse @stackman27
+x/tokenfactory/**/* @mattverse @nicolaslara @p0mvn
+x/twap/**/* @nicolaslara @p0mvn
+x/txfees/**/* @nicolaslara @p0mvn
+x/valset-pref/**/* @mattverse @stackman27
+
```

Comment:
Could we add also:

`/.github/ @osmosis-labs/devops` 


Should we also add the other folders in the root repo?

```
# e.g.
/cosmwasm/ ... 
/wasmbinding/ ...
```

Reply:
> Should we also add the other folders in the root repo?

For other folders in root repo, we do not have a claer code owner set yet, I think we can leave it until further discussion has happend for the ownership

Reply:
> Could we add also:
> 
> /.github/ @osmosis-labs/devops

Sure, to which directories should I be adding the devops team?

Reply:
For the @devops team let's add:

/.github/ @osmosis-labs/devops
Dockerfile @osmosis-labs/devops
.dockerignore @osmosis-labs/devops 
Makefile @osmosis-labs/devops
/tests/localosmosis/ @osmosis-labs/devops
/tests/localrelayer/ @osmosis-labs/devops

Reply:
done